### PR TITLE
Sc switch

### PR DESCRIPTION
**Pull request by meguiraun**
A small component to allow SC on/off from the data collection view.
It comes from the workshop training

### CODE REVIEW

File: mxcube3/ui/containers/BeamlineSetupContainer.jsx [hunk 1]
```diff
@@ -4,7 +4,8 @@ import { connect } from 'react-redux';
 import { Row, Col, Table, Popover } from 'react-bootstrap';
 import PopInput from '../components/PopInput/PopInput';
 import BeamlineActions from './BeamlineActionsContainer';
-import InOutSwitch from '../components/InOutSwitch/InOutSwitch';
+import InOutSwitch2 from '../components/InOutSwitch2/InOutSwitch2';
```
Comment (marcus-oscarsson) [inline mxcube3/ui/containers/BeamlineSetupContainer.jsx]: InOutSwitch2 and InOutSwitch have been merged to InOutSwitch. The idea was to only use InOutSwitch

File: mxcube3/ui/containers/BeamlineSetupContainer.jsx [hunk 2]
```diff
@@ -269,12 +271,10 @@ class BeamlineSetupContainer extends React.Component {
               </Col>
               {this.createActuatorComponent()}
               <Col sm={2} className="pull-right">
-                <LabeledValue
-                  suffix=""
-                  name="Sample changer"
-                  value={this.props.sampleChanger.state}
-                  level={this.props.sampleChanger.state === 'READY' ? 'info' : 'warning'}
-                  look={'vertical'}
+                <SampleChangerSwitch
```
Comment (marcus-oscarsson) [inline mxcube3/ui/containers/BeamlineSetupContainer.jsx]: Does this also show the state of the sample changer or only if its on or off ?

**Comment by marcus-oscarsson** [comment]
Could you take a closer look at why Travis does not pass ?, there seems to be some issue with "npm run build"

**Comment by meguiraun** [comment]
yes, I will. Probably because the InOutSwitch, strange I did not see locally... on it.

**Comment by marcus-oscarsson** [comment]
Thanks Mikel !

**Comment by meguiraun** [comment]
It has been a while since I last run master... more problems that expected :)

File: mxcube3/ui/containers/BeamlineSetupContainer.jsx (hunk 2 above)
Comment (meguiraun) [reply mxcube3/ui/containers/BeamlineSetupContainer.jsx]: It shows state, same functionality as with the shutters
image

Comment (marcus-oscarsson) [reply mxcube3/ui/containers/BeamlineSetupContainer.jsx]: Ok, great :)

**Pull request merged**